src/components/WeeklyCalendar/readme.md

```diff
@@ -303,3 +303,130 @@ const WeeklyCalendarExample = () => {
+        endDate: new Date(daysOfWeek[0].setHours(6,30, 0, 0)),
+        backgroundColor: 'rgba(253,230,230,1)',
+        color: 'rgba(254,72,73,1)',
+    },
+    {
+        id: 2,
+        title: 'JL Torres',
+        startDate: new Date(daysOfWeek[0].setHours(7, 30, 0, 0)),
+        endDate: new Date(daysOfWeek[0].setHours(8, 0, 0, 0)),
+        backgroundColor: 'rgba(255,204,0,0.4)',
+        color: 'rgba(255,157,0,1)',
+    },
+    {
+        id: 3,
+        title: 'Leandro Torres',
+        startDate: new Date(daysOfWeek[0].setHours(11, 0, 0, 0)),
+        endDate: new Date(daysOfWeek[0].setHours(12, 15, 0, 0)),
+        backgroundColor: 'rgba(145,220,193,1)',
+        color: 'rgba(0,171,142,1)',
+    },
+    {
+        id: 4,
+        title: 'Yuri V. Munayev',
+        startDate: new Date(daysOfWeek[1].setHours(6, 30, 0, 0)),
+        endDate: new Date(daysOfWeek[1].setHours(7, 30, 0, 0)),
+        backgroundColor: 'rgba(254,72,73,1)',
+    },
+    {
+        id: 5,
+        title: 'Tahimi',
+        startDate: new Date(daysOfWeek[1].setHours(8, 0, 0, 0)),
+        endDate: new Date(daysOfWeek[1].setHours(8, 15, 0, 0)),
+        backgroundColor: 'rgba(240,243,56,1)',
+    },
+    {
+        id: 6,
+        title: 'Tahimi L',
+        startDate: new Date(daysOfWeek[2].setHours(8, 0, 0, 0)),
+        endDate: new Date(daysOfWeek[2].setHours(9, 30, 0, 0)),
+        backgroundColor: 'rgba(240,243,56,1)',
+    },
+    {
+        id: 7,
+        title: 'Sara',
+        startDate: new Date(daysOfWeek[3].setHours(6, 0, 0, 0)),
+        endDate: new Date(daysOfWeek[3].setHours(6, 30, 0, 0)),
+        backgroundColor: 'rgba(254,72,73,1)',
+    },
+    {
+        id: 8,
+        title: 'Tahimi',
+        startDate: new Date(daysOfWeek[3].setHours(6, 30, 0, 0)),
+        endDate: new Date(daysOfWeek[3].setHours(7, 0, 0, 0)),
+        backgroundColor: 'rgba(254,72,73,1)',
+    },
+    {
+        id: 9,
+        title: 'Reinier',
+        startDate: new Date(daysOfWeek[3].setHours(7, 30, 0, 0)),
+        endDate: new Date(daysOfWeek[3].setHours(8, 15, 0, 0)),
+        backgroundColor: 'rgba(255,204,0,1)',
+    },
+    {
+        id: 10,
+        title: 'Sara P',
+        startDate: new Date(daysOfWeek[4].setHours(6, 30, 0, 0)),
+        endDate: new Date(daysOfWeek[4].setHours(8, 0, 0, 0)),
+        backgroundColor: 'rgba(254,72,73,1)',
+    },
+    {
+        id: 11,
+        title: 'Leo Torres',
+        startDate: new Date(daysOfWeek[5].setHours(6, 0, 0, 0)),
+        endDate: new Date(daysOfWeek[5].setHours(7, 0, 0, 0)),
+        backgroundColor: 'rgba(254,72,73,1)',
+
+    },
+    {
+        id: 12,
+        title: 'Tahimi',
+        startDate: new Date(daysOfWeek[6].setHours(8, 0, 0, 0)),
+        endDate: new Date(daysOfWeek[6].setHours(9, 30, 0, 0)),
+        backgroundColor: 'rgba(240,243,56,1)',
+    },
+];
+
+const StyledCard = styled(Card)`
+    height: 600px;
+    padding: 1rem;
+`;
+const initialState = {
+    currentWeek: new Date(),
+};
```

Comment:
Let's use a function component with hooks for this, note that if we copy and paste this example in order to use it, it will not work. I know there are a bunch of examples in this way but new ones need to be done in the other way:
```
function CustomWeeklyCalendar() {
...
}

    <CustomWeeklyCalendar />
```